<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelBrowser</class>
 <widget class="QWidget" name="ModelBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMTK Model Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="modelTree"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addGroupButton">
       <property name="text">
        <string>Add Group</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addToGroupButton">
       <property name="text">
        <string>Add To Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeFromGroupButton">
       <property name="text">
        <string>Remove From Group</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
